tests/test_ai.py

```diff
@@ -0,0 +1,206 @@
+"""Unit tests for hass_nabucasa.llm."""
+
+from __future__ import annotations
+
+import io
+from types import SimpleNamespace
+from unittest.mock import AsyncMock, MagicMock, patch
+
+from litellm.exceptions import (
+    APIError,
+    AuthenticationError,
+    RateLimitError,
+    ServiceUnavailableError,
+)
+import pytest
+
+import hass_nabucasa.llm as ai_module
+from hass_nabucasa.llm import (
+    LLM,
+    LLMAuthenticationError,
+    LLMImageAttachment,
+    LLMRateLimitError,
+    LLMServiceError,
+)
+
+
+def _mock_llm() -> LLM:
+    cloud = MagicMock(valid_subscription=True)
+    llm = LLM(cloud)
+    llm._token = "token"
+    llm._base_url = "https://api.example"
+    llm._generate_data_model = "responses-model"
+    llm._generate_image_model = "image-model"
+    llm.async_ensure_token = AsyncMock()
+    return llm
```

Comment:
Use the cloud fixture defined in conftest instead of this local one.
(like how it is done for other tests, like test_coice, tests_account_api and others)